app/views/katello/api/v2/content_facet/show.json.rabl

```diff
@@ -40,6 +40,9 @@ child :content_facet => :content_facet_attributes do
     node(:view_products) { user.can?("view_products") }
     node(:create_bookmarks) { user.can?("create_bookmarks") }
   end
+
+  attributes :bootc_booted_image, :bootc_booted_digest, :bootc_available_image, :bootc_available_digest,
```

Comment:
This isn't needed anymore in show rabl since it already extends from base.json.rabl..

Reply:
Fixed!